spec/openapi.yaml

```diff
@@ -68,6 +81,124 @@ components:
         - text
         - firmware
 
+    FileInfo:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "PRINT_FILE"
+        refs:
+          type: object
+          properties:
+            download:
+              type: string
+              example: "/api/files/local/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode/raw"
+            icon:
+              type: string
+              example: "/api/thumbnails/local/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode.small.png"
+            thumbnail:
+              type: string
+              example: "/api/thumbnails/local/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode.orig.png"
+        size:
+          type: integer
+          example: 424242
+        path:
+          type: string
+          example: "/local/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode"
+        name:
+          type: string
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        creation_timestamp:
+          type: integer
+          example: 1648038843
```

Comment:
What are the units?

Reply:
Its time stamp, which is part of attribute name. May be question is, can you create timestamp in Buddy firmware?